Xamarin.Essentials/FileSystem/FileSystem.shared.cs

```diff
@@ -73,7 +73,9 @@ internal string GetContentType()
 
             // we haven't been able to determine this
             // leave it up to the sender
-            return null;
+            // return null;
+            // Instead of returning null return a catch all mime-type - see #1225
```

Comment:
don't like to put in issue numbers in the code, as we have blame to help us out and link back.

Do we want it to be octet-stream? does everythign support that?

Reply:
Fair enough ref comments,  octet-stream was suggested in the comments and is as generic as it comes, there are no real application handlers for it so gets attached / downloaded and up to the end user to handle

Reply:
I think if we just test it out on iOS/Android that then would be good... just want to does work... we can just document it and recommend people set it manually.